### PR TITLE
Button-Title-Case & Removed-White-Background

### PR DESCRIPTION
**Pull request by aktersabina122**
### What does this PR do?

This PR Changed all the button into title case and removed white background from -tertiary class

### Any background context you want to provide?

Screen Shot 2019-07-03 at 5 21 30 PM

### What are the relevant tickets/cards?

Refs [Pivotal ID #66849537]()
https://www.pivotaltracker.com/n/projects/2019313/stories/166849537

### Checklist

* [] ### **Documentation added for new features/changed endpoints.**
* [x] Added appropriate feature/unit tests.
* [x ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: resources/assets/components/utilities/Button/button.scss [hunk 1]
```diff
@@ -32,5 +34,5 @@
 }
 
 .button.-tertiary {
-  background: #fff;
+  background: none;
```
Comment (mendelB) [inline resources/assets/components/utilities/Button/button.scss]: I think we might be able to just get rid of this entire `.button.-tertiary` rule completely. Unless there's some other Forge rule that we're overriding? (When you inspect the element, do you see some other rule adding a background other than the default?)

Comment (aktersabina122) [reply resources/assets/components/utilities/Button/button.scss]: Let me check. Thanks

Comment (aktersabina122) [reply resources/assets/components/utilities/Button/button.scss]: > I think we might be able to just get rid of this entire `.button.-tertiary` rule completely. Unless there's some other Forge rule that we're overriding? (When you inspect the element, do you see some other rule adding a background other than the default?)

No, I do not see any other rule adding a background other than the default. Do you want me to remove the .button.-tertiary class?

Comment (mendelB) [reply resources/assets/components/utilities/Button/button.scss]: Ok perfect! Then yes we can get rid of the whole rule 🗑

Comment (aktersabina122) [reply resources/assets/components/utilities/Button/button.scss]: I removed the .button.-tertiary class from the button.scss. Please let me know if everything looks good. Thanks

**Pull request merged**